### DISCUSSION TITLE
WorldGuard Extra Flags

### DISCUSSION THREAD

**Discussion by LeeTheTech** [Zrips/CMI#1138] [Enhancement]
Could you please add support for respawn-location for: 
https://www.spigotmc.org/resources/worldguard-extra-flags.4823/

I'd like to be able to set a region priority in the cmi config.

So, if player dies in region they respawn at the region respawn location instead of the spawn, set by cmi.

**Comment by mrfloris** [comment]
I've seen a couple of discord requests lately, including today. Bumping :)

**Reply by R4z0rX** [reply]
Re-bumping...
